src/components/Typography/Typography.style.ts

```diff
@@ -126,5 +152,14 @@ export const typographyWrapper =
       body03,
     };
 
-    return allStyles[type];
+    const textColorCategory = isInverted ? 'inverted' : 'light';
+    const textColor = `${textColorCategory}.${type}` as DotKeys<typeof textColorFigma>;
+
+    return css`
+      ${allStyles[variant]};
+      font-style: ${isItalic ? 'italic' : undefined};
+      font-weight: ${isBold ? 'bold' : undefined};
+      text-decoration: ${isUnderline ? 'underline' : undefined};
```

Comment:
_[optional]_ same here. we have textDecoration tokens for that. pretty much the same, but we can do it for consistency reasons.

Reply:
fixed here [`03c7ba6` (#609)](https://github.com/Orfium/orfium-ictinus/pull/609/commits/03c7ba6f8d3f4a3eb76fdc6e2c401766f55c50c6)